osu.Game/Utils/FormatUtils.cs

```diff
@@ -12,14 +12,14 @@ public static class FormatUtils
         /// </summary>
         /// <param name="accuracy">The accuracy to be formatted</param>
         /// <returns>formatted accuracy in percentage</returns>
-        public static string FormatAccuracy(this double accuracy) => $"{accuracy:0.00%}";
+        public static string FormatAccuracy(this double accuracy)
+        {
+            // we don't ever want to show 100% when the accuracy is below perfect, even if it rounds to 100%.
+            if (accuracy < 1 && accuracy > 0.9999)
+                accuracy = 0.9999;
 
-        /// <summary>
-        /// Turns the provided accuracy into a percentage with 2 decimal places.
-        /// </summary>
-        /// <param name="accuracy">The accuracy to be formatted</param>
-        /// <returns>formatted accuracy in percentage</returns>
-        public static string FormatAccuracy(this decimal accuracy) => $"{accuracy:0.00}%";
+            return $"{accuracy:0.00%}";
```

Comment:
With the way this method is written, the test cases attached in this PR fail on systems that use a different decimal separator than the full stop due to regional settings (i.e. mine, as Polish uses a decimal comma rather than a full stop).

Probably need to either rewrite the test to format both values according to culture settings, or to put a `FormattableString.Invariant()` call around this, to avoid confusing others as to why those tests are failing.

Reply:
Considered this from a CI perspective, but not from other-users-running-tests.